presto-main/src/main/java/com/facebook/presto/execution/SqlQueryExecution.java

```diff
@@ -325,6 +325,12 @@ public BasicQueryInfo getBasicQueryInfo()
                 .orElseGet(() -> stateMachine.getBasicQueryInfo(Optional.ofNullable(queryScheduler.get()).map(SqlQueryScheduler::getBasicStageStats)));
     }
 
+    @Override
+    public int getRunningTask()
```

Comment:
This returns a running task count, but from the name it looks like it should return a single discrete task. You could go either way naming convention wise. I don't feel strongly.